### PR TITLE
[14.0][ADD] stock_move_delay_report

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Forward port of #270 

Supersedes https://github.com/OCA/stock-logistics-workflow/pull/1233

@ForgeFlow

### CODE REVIEW

File: stock_move_delay_report/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    def _compute_date_delay(self):
+        for rec in self:
+            if rec.state == "done" and rec.date_deadline:
+                rec.date_delay = (rec.date - rec.date_deadline).days
+            elif not rec.date_deadline:
+                rec.date_delay = 0
+            else:
+                rec.date_delay = None
+
+    def _compute_responsible(self):
```
Comment (rousseldenis) [inline stock_move_delay_report/models/stock_move.py:35]: @DavidJForgeFlow You should add depends

File: stock_move_delay_report/report/delay_report.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            sm.id AS move_id,
+            sm.product_id,
+            sm.reference,
+            sm.location_id AS location_src_id,
+            sm.location_dest_id,
+            sm.date_delay,
+            %s,
+            sm.product_uom,
+            sm.responsible_id
+        """ % (
```
Comment (rousseldenis) [inline stock_move_delay_report/report/delay_report.py:41]: You cannot pass query parameters like that.

File: stock_move_delay_report/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module adds a report to check the delay from incoming and outgoing moves grouped by Responsible Partner.
```
Comment (rousseldenis) [inline stock_move_delay_report/readme/DESCRIPTION.rst]: Could you elaborate a little bit more on what is the delay exactly ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
First code review

File: stock_move_delay_report/report/delay_report.py (hunk 2 above)
Comment (DavidJForgeFlow) [reply stock_move_delay_report/report/delay_report.py:41]: @rousseldenis what do you mean? Why I can't do it, please explain me if you can

File: stock_move_delay_report/readme/DESCRIPTION.rst (hunk 3 above)
Comment (DavidJForgeFlow) [reply stock_move_delay_report/readme/DESCRIPTION.rst]: Readme updated, btw, delay is the difference in days between the deadline and the effective date

File: stock_move_delay_report/report/delay_report.py (hunk 2 above)
Comment (rousseldenis) [reply stock_move_delay_report/report/delay_report.py:41]: @DavidJForgeFlow  Query parameters should be passed to execute as arguments not as formatting.

https://github.com/odoo/odoo/blob/14.0/odoo/sql_db.py#L293

Many examples in core.

**Pull request closed without merging**